pages/blog/[slug].vue

```diff
@@ -171,7 +170,19 @@ useHead({
             />
           </template>
         </CardMeta>
+
+        <h3 class="about-the-author">
+          About the Author
+        </h3>
+
+        <RichText
+          v-if="page?.aboutTheAuthor"
+          data-test="aboutTheAuthor"
```

Comment:
@jendiamond What is this `data-test` attribute for?

Reply:
Data-test is for the cypress tests